### PR TITLE
feat: member can also invite by SSB-ID

### PR DESCRIPTION
**Pull request by decentral1se**
Closes https://github.com/ssb-ngi-pointer/go-ssb-room/issues/283. Follows from https://github.com/ssb-ngi-pointer/go-ssb-room/pull/179.

### CODE REVIEW

**Comment by decentral1se** [comment]
I am not sure the [CI failure](https://github.com/ssb-ngi-pointer/go-ssb-room/runs/5960017371?check_suite_focus=true) is related to my change? I don't quite understand what it is trying to do but maybe it doesn't like me rebasing/force pushing on the same branch? Or is this some permissions issue? :woman_shrugging: :see_no_evil:

File: web/handlers/admin/members_test.go [hunk 1]
```diff
@@ -112,12 +112,12 @@ func TestMembersAdd(t *testing.T) {
...
 	checkInputsAreDisabled(false)
 
-	// verify that inputs are disabled for RoleMember
+	// verify that inputs are enabled for RoleMember
 	ts.User = roomdb.Member{
 		ID:   7331,
 		Role: roomdb.RoleMember,
 	}
-	checkInputsAreDisabled(true)
+	checkInputsAreDisabled(false)
```
Comment (quickdudley) [inline web/handlers/admin/members_test.go:120]: That's a pretty counterintuitive way to check that inputs are enabled.

Comment (decentral1se) [reply web/handlers/admin/members_test.go:120]: Can see why you might say that but then it is done several times in the rest of the test suite so I just do the same :woman_shrugging:

**Comment by decentral1se** [comment]
Bump, anyone got cycles to review / merge. It is a small one :crossed_fingers:

**Comment by staltz** [comment]
Thank you @decentral1se

**Pull request merged**